lib/App.js

```diff
@@ -3378,6 +3378,15 @@ You can create an API Key on https://platform.openai.com/api-keys.`);
           if (fileStats.isFile() && filePath.endsWith('.json')) {
             if (file.startsWith('.')) return;
 
+            // Special handling for driver files
+            if (dir.includes('drivers')) {
```

Comment:
This will prevent translation of all other compose files in `.homecompose/drivers`?

https://apps.developer.homey.app/advanced/homey-compose